app/svelte.config.js

```diff
@@ -1,6 +1,16 @@
 import { vitePreprocess } from '@sveltejs/kit/vite';
 import adapter from '@sveltejs/adapter-node';
 import { resolve } from 'path';
+
+function formatURLForCsp(envVar) {
+	if (!envVar) return;
+	const url = new URL(envVar);
```

Comment:
Le nom `envVar` me paraît trop spécifique.

```suggestion
function formatURLForCsp(originalUrl) {
	if (!originalUrl) return;
	const url = new URL(originalUrl);
```